ci/docker/x86_64-unknown-uefi/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM ubuntu:18.04
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+    gcc libc6-dev ca-certificates
```

Comment:
Is this intended to be hooked up to CI? (it doesn't happen automatically, extra configuration would be needed)

Reply:
@oli-obk can we do CI configs in a follow up PR, so that this fix can get merged?